### PR TITLE
Battling with different number of ships. Fixes #3.

### LINKED ISSUE #3: Multiship fleets
Refer to #1 for fleet details.
Initial fleets should add one ship each (totalling to two ships of the same design per each fleet).

No advanced target selection algorithm required.

### PR DESCRIPTION
**Pull request by octocrux**

### CODE REVIEW

File: first.js [hunk 1]
```diff
@@ -1,48 +1,79 @@
...
+  var simulations = 20000;
   var i;
 
-  var battleSimulation = function () {
-    var attacker = {alive: true, initiative: 1, hit: (5 / 6)};
-    var defender = {alive: true, initiative: 0, hit: (4 / 6)};
-    var first;
-    var second;
-    var success = false;
+  var Ship = function (ship) {
```
Comment (marvelousNinja) [inline first.js]: It's kind of.. Fleet isn't it?

File: first.js [hunk 2]
```diff
@@ -1,48 +1,79 @@
...
+    this.initiative = ship.initiative;
+    this.hit = ship.hit;
+    this.count = ship.count;
+
+    return this;
```
Comment (marvelousNinja) [inline first.js]: It's not required to return this from that method.

**Comment by octocrux** [comment]
I hate this dog.

File: first.js [hunk 3]
```diff
@@ -1,48 +1,79 @@
...
+      firstToHit = defender;
+      secondToHit = attacker;
     }
 
-    while (attacker.alive && defender.alive) {
+    while (attacker.count > 0 && defender.count > 0) {
+      successfulHits = 0;
+      for (h = 0; h < firstToHit.count; h++) {
+        success = Math.random() >= firstToHit.hit;
+        if (success) {
```
Comment (marvelousNinja) [inline first.js]: It can be shorter like ```if (Math.random() >= firstToHit.hit)``` and etc.

File: first.js [hunk 4]
```diff
@@ -1,48 +1,79 @@
...
+      }
 
-      success = Math.random() >= first.hit;
-      if (success) {
-        second.alive = false;
-        break;
+      successfulHits = 0;
+      for (h = 0; h < secondToHit.count; h++) {
+        success = Math.random() >= secondToHit.hit;
+        if (success) {
```
Comment (marvelousNinja) [inline first.js]: Same as above.

**Comment by marvelousNinja** [comment]
Good to go.

**Pull request merged**